src/main/java/com/paneedah/mwc/entities/BanditEntityFactory.java

```diff
@@ -29,9 +29,10 @@ public void createEntity(ModContext modContext) {
         .withMaxHealth(40)
         .withEntityIdSupplier(() -> 10000)
         .withEquipmentOption(Guns.VSSVintorez, EnumDifficulty.EASY, 0.1f, Magazines.VSSVintorezMag)
+        .withEquipmentOption(Guns.VSSVintorez, EnumDifficulty.HARD, 0.1f, Magazines.ASValMag)
         .withEquipmentOption(Guns.MakarovPM, EnumDifficulty.EASY, 0.1f, Magazines.MakarovMag)
         .withEquipmentOption(Guns.Remington870, EnumDifficulty.EASY, 0.07f)
-        .withPrimaryEquipmentDropChance(0.4f)
+        .withPrimaryEquipmentDropChance(1.0f)
```

Comment:
That is WAY to op, same please revert, I would say even the current setting is too op